pallets/interest-accrual/src/lib.rs

```diff
@@ -51,6 +51,74 @@
 //!            nd   = 10      nd   = 10 + (20 / 1.5) = 23.33
 //!            debt = 10      debt = 35
 //! ```
+//!
+//! ## Basics of shared rate accrual and "normalized debts"
+//!
+//! When we want to compute the interest accrued on some value, the
+//! high-level equation is:
+//!
+//! ```ignore
+//! rate_per_second.pow(debt_age) * debt_base_value
+//! ```
+//!
+//! Computing that pow for everything is expensive, so we want to only
+//! do it once for any given `rate_per_second` and share that result
+//! across multiple loans. Because these loans might not have been
+//! created at the same time as each other (or the rate), we must
```

Comment:
```suggestion
//! do it once for any given `rate_per_second` and share that result
//! across multiple debts. Because these debts might not have been
//! created at the same time as each other (or the rate), we must
```